### PR TITLE
Adding a simple accordion & Tab for an example to codesandbox

### PR DESCRIPTION
**Pull request by britt6612**
I have added in two components that were missing from codesandbox. 
Tabs and Accordion with a simple example for both.

### CODE REVIEW

File: src/Accordion.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+export default () => (
+  <SandboxComponent>
+    <Box width='100%'>
+      <Accordion animate='true' multiple='true' margin='small'>
+        <AccordionPanel label='Panel 1'>
+          <Box background='light-1'>Panel 1 content</Box>
+        </AccordionPanel>
+        <AccordionPanel label='Panel 2'>
+          <Box style={{ height: '200px' }} background='light-1'>Panel 2 content</Box>
```
Comment (ericsoderberghp) [inline src/Accordion.js]: We should avoid inline `style` and instead use something like `height="small"`.

File: src/Accordion.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Accordion, AccordionPanel, Box } from 'grommet';
+import SandboxComponent from './SandboxComponent';
+
+export default () => (
+  <SandboxComponent>
+    <Box width='100%'>
```
Comment (ericsoderberghp) [inline src/Accordion.js]: `fill="horizontal"` would be the grommet way to do this.

File: src/Accordion.js (hunk 1 above)
Comment (britt6612) [reply src/Accordion.js]: Thank you!

**Comment by ShimiSun** [comment]
fixes https://github.com/grommet/grommet/issues/2647

**Pull request merged**